src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Let's separate your solution into three private methods.
One for reading data from file.
One for writing data to the file.
And one for statistic generation.

Such core will be more readable, reusable and easy to support